packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -106,10 +106,15 @@ function build(
     // if the vref corresponds to a Remotable, keep a strong reference to it
     // until the kernel tells us to release it
     const { type, allocatedByVat, virtual } = parseVatSlot(vref);
-    if (type === 'object' && allocatedByVat && !virtual) {
-      const remotable = slotToVal.get(vref).deref();
-      assert(remotable, X`somehow lost Remotable for ${vref}`);
-      exportedRemotables.add(remotable);
+    if (type === 'object' && allocatedByVat) {
```

Comment:
It's true that it's no longer just about remotables, so a name change is warranted.  However, I don't like `exportVrefToKernel` since this function doesn't do any exporting but rather takes note that something has been exported.  I will come up with a better name.

Reply:
How does `retainExportedVref` grab you?

Reply:
Yeah, that works.. it's specifically about ensuring the `exported` flag is `true`, and handling both the false-true and the true-true transitions. I never know what verb to use for that: `setExported` might be correct, if paired with `clearExported`, but for me `set` frequently implies that we know the value was `false` beforehand (especially when `set` is paired with `reset`).

Reply:
One of my earliest engineering mentors (Jim Cutler, a EE who was my boss at the UofM Space Physics Lab) was very insistent that one must always pair "set" with "clear" and NEVER EVER pair "set" with "reset".  Over lunch or something I can share the story of how the latter lead to the loss of a multimillion dollar spacecraft.  However, we also use "set" to mean "assign", which is the meaning in this context.  We probably would like to have an alternative verb which is not as clunky as "assign" but less prone to confusion than "set".